COMMERCE-778 Remove deleted module reference

ZoltanTakacs opened the pull request:
https://issues.liferay.com/browse/COMMERCE-778

```
Dependency in modules/apps/commerce/commerce-test/build.gradle points to nonexistent project directory modules/apps/commerce/commerce-organization-test-util
```

ZoltanTakacs committed:
COMMERCE-778 Remove module deleted module reference

ZoltanTakacs committed:
COMMERCE-778 Auto SF

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

ZoltanTakacs commented:
ci:test
/cc @marco-leo @ethanbustad currently we have a gradle issue (see https://github.com/liferay/com-liferay-commerce/pull/383) because we removed the organization related modules from the branch and one reference was still made to the test module. I've removed it and run SF.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/53/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-6.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/53/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Validation FAILED.</h1><p>Build Time: 8 minutes 49 seconds 801 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/25d8e3eeb0f745436911a3e21429bf6c5674cff4">25d8e3eeb0f745436911a3e21429bf6c5674cff4</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :x:<pre><code>     [echo] * What went wrong:
     [echo] Execution failed for task ':apps:commerce:commerce-api:baseline'.
     [echo] &gt; Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal-7.1.x/tools/sdk/dist/com.liferay.commerce.api-4.0.0.jar against /opt/dev/projects/github/liferay-portal-7.1.x/.gradle/caches/modules-2/files-2.1/com.liferay.commerce/com.liferay.commerce.api/3.1.0/13f17ebab47c3e9778ee0c43375f5e91f3fbe245/com.liferay.commerce.api-3.1.0.jar
     [echo] 
     [echo] * Try:
     [echo] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [echo] 
     [echo] * Exception is:
     [echo] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':apps:commerce:commerce-api:baseline'.
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [echo] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [echo] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
     [echo] 	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
     [echo] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)</code></pre></li><li>central-requirements-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-21.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/38//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>7 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-21.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/38//testReport">here</a>.</h5></div></html>

ZoltanTakacs committed:
COMMERCE-778 Run baseline

ZoltanTakacs commented:
ci:retest

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-21.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/56/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-21.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/56/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Validation FAILED.</h1><p>Build Time: 28 minutes 19 seconds 562 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/25d8e3eeb0f745436911a3e21429bf6c5674cff4">25d8e3eeb0f745436911a3e21429bf6c5674cff4</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :x:<pre><code>     [echo] java.lang.Exception: Found 4 formatting issues:
     [echo] 1: Reference 'Group' not needed for Entity 'CommerceAccountUserRel': ./commerce-account-service/service.xml (SourceCheck:XMLServiceReferenceCheck)
     [echo] 2: Illegal call to class 'com.liferay.portal.kernel.service.persistence.GroupPersistence', see LPS-68923: ./commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountLocalServiceImpl.java 254 (Checkstyle:PersistenceCallCheck)
     [echo] 3: Do not use type 'ThemeDisplay' in API method signature, see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/documentation/api_method_signatures.markdown: ./commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/order/CommerceOrderResource.java (SourceCheck:JavaAPISignatureCheck)
     [echo] 4: File must start with copyright: ./commerce-organization-order-web/src/main/resources/META-INF/resources/edit_order_details.jsp (SourceCheck:CopyrightCheck)
     [echo] 
     [echo] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:414)
     [echo] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:241)
     [echo] 
     [echo] FAILURE: Build failed with an exception.
     [echo] 
     [echo] * What went wrong:
     [echo] Execution failed for task ':checkSourceFormatting'.
     [echo] &gt; Process 'command '/opt/java/jdk1.8.0_121-x64/bin/java'' finished with non-zero exit value 1
     [echo] 
     [echo] * Try:
     [echo] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [echo] 
     [echo] * Exception is:
     [echo] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':checkSourceFormatting'.
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [echo] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)</code></pre></li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-20.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/41//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-20.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/41//testReport">here</a>.</h5></div></html>

ethanbustad commented:
@ZoltanTakacs I sent a PR here https://github.com/liferay/com-liferay-commerce/pull/388 with fixes for the remaining SF issues.
